testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHivePropertiesTable.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.tests.product.hive;
+
+import org.testng.annotations.Test;
+
+import static io.trino.tempto.assertions.QueryAssert.assertThat;
+import static io.trino.tempto.query.QueryExecutor.query;
+import static io.trino.tests.product.TestGroups.HIVE_VIEWS;
+import static io.trino.tests.product.utils.QueryExecutors.onHive;
+import static io.trino.tests.product.utils.QueryExecutors.onTrino;
+
+public class TestHivePropertiesTable
+        extends HiveProductTest
+{
+    @Test
+    public void testTrinoViewPropertiesTable()
+    {
+        onTrino().executeQuery("DROP TABLE IF EXISTS test_trino_view_properties_base");
+        onTrino().executeQuery("DROP VIEW IF EXISTS test_trino_view_properties");
+        onTrino().executeQuery("CREATE TABLE test_trino_view_properties_base (col INT)");
+        onTrino().executeQuery("CREATE VIEW test_trino_view_properties AS SELECT * FROM test_trino_view_properties_base");
+
+        assertThat(query("SELECT * FROM \"test_trino_view_properties$properties\""))
+                .hasRowsCount(1);
+
+        onTrino().executeQuery("DROP VIEW IF EXISTS test_trino_view_properties");
+        onTrino().executeQuery("DROP TABLE IF EXISTS test_trino_view_properties_base");
+    }
+
+    @Test(groups = HIVE_VIEWS)
+    public void testHiveViewPropertiesTable()
+    {
+        onTrino().executeQuery("DROP TABLE IF EXISTS test_hive_view_properties_base");
+        onTrino().executeQuery("DROP VIEW IF EXISTS test_hive_view_properties");
+        onTrino().executeQuery("CREATE TABLE test_hive_view_properties_base (col INT)");
+        onHive().executeQuery("CREATE VIEW test_hive_view_properties AS SELECT * FROM test_hive_view_properties_base");
+
+        assertThat(query("SELECT * FROM \"test_hive_view_properties$properties\""))
+                .hasRowsCount(1);
```

Comment:
Do all Hive distributions only set just the timestamp as the property? Maybe the assertion needs to be more explicit?